### ISSUE TITLE
Allow master to advertise for events

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1624]
Currently, if the community wants to prepare an event, or if there is a new release, master have no way to communicate about it.
I think such feature would help gathering a more active community, without relying on pulling infos from the forum or IRC or whatever channel, but to inform player which just log in a server. Maybe through the updater?

**Comment by Gireen** [comment]
Blog posts are displayed in the updater. 
With master you mean the masterserver? It can already send announcements if clients query for them. But the functionality in the client is not implemented.

**Comment by bmorel** [comment]
Right, there's the updater. I forgot about it, since there's rarely news.

> With master you mean the masterserver?

Yes.

> It can already send announcements if clients query for them. But the functionality in the client is not implemented.

Good to know.

**Comment by slipher** [comment]
The updater isn't suitable for this in its current form, because you only see the news screen when the game requires a software update.

**Comment by cu-kai** [comment]
Trtemulous used to fetch "news" from the internet and save it to a local file. Can we not do something similar? (Can also use this for "events" etc).

See: https://github.com/darklegion/tremulous/blob/c862a5340c8de44dcc7abaff170c20c04f9340e8/src/client/cl_ui.c#L89
